src/frontend/js/components/CourseProductCourseRuns/EnrollableCourseRunList.tsx

```diff
@@ -61,7 +66,9 @@ const EnrollableCourseRunList = ({ courseRuns, order }: Props) => {
   const formRef = useRef<HTMLFormElement>(null);
 
   const [selectedCourseRun, setSelectedCourseRun] = useState<Maybe<Joanie.CourseRun>>();
+  const [errors, setErrors] = useState<MessageDescriptor[]>([]);
```

Comment:
I think we should update the `useEnrollment` hook to base it from `useResource`. In this way, we are able to retrieve an error message on get / create / update / delete enrollment.